src/pharmpy/modeling/summarize_individuals.py

```diff
@@ -132,8 +132,14 @@ def ofv(model: Model) -> Union[Series, float]:
     return np.nan if res is None else res.individual_ofv
 
 
-def dofv(parent_model: Union[Model, None], candidate_model: Model) -> Union[Series, float]:
-    return np.nan if parent_model is None else ofv(parent_model) - ofv(candidate_model)
+def dofv(
+    parent_model: Union[Model, None], candidate_model: Union[Model, None]
+) -> Union[Series, float]:
+    return (
+        np.nan
+        if parent_model is None or candidate_model is None
+        else ofv(parent_model) - ofv(candidate_model)
+    )
```

Comment:
LGTM!

Reply:
Actually, if this is meant to fix #1101 I think the problem comes from `model.modelfit_results.individual_ofv` being `None` instead of `np.nan`.